examples/gno.land/r/gov/dao/v3/impl/govdao_test.gno

```diff
@@ -275,6 +277,42 @@ func TestUpgradeDaoImplementation(t *testing.T) {
 	urequire.Equal(t, true, contains(dao.Render("7"), "YES PERCENT: 68.42105263157895%"))
 }
 
+func TestNonMemberProposalFee(cur realm, t *testing.T) {
+	loadMembers()
+
+	portfolio := "# This is my portfolio:\n\n- THINGS"
+	nm1 := testutils.TestAddress("nm1")
+
+	testing.SetOriginCaller(noMember)
+	testing.SetRealm(testing.NewCodeRealm("gno.land/r/gov/dao/v3/impl"))
+
+	// Test: Non-member without fee should fail with realm validation error
+	uassert.AbortsContains(t, "proposal creation must be done directly by a user or through the r/gov/dao proxy", func(cur realm) {
+		dao.MustCreateProposal(cross, NewAddMemberRequest(cur, nm1, memberstore.T2, portfolio))
+	})
+
+	// Test: Non-member with insufficient fee should also fail with realm validation error
+	testing.SetOriginSend(chain.Coins{chain.Coin{Denom: "ugnot", Amount: 500000}}) // 0.5 GNOT - insufficient
+	uassert.AbortsContains(t, "proposal creation must be done directly by a user or through the r/gov/dao proxy", func(cur realm) {
```

Comment:
why the error here should say "proposal creation must be done directly by a user or through the r/gov/dao proxy", i would expect the test to ensure there is a "insufficient fee, user must send 1_000_000 gnot" or smth like this, same for all your test in this function.